tests/std/tests/P0220R1_optional/test.cpp

```diff
@@ -5718,7 +5718,7 @@ class X
 {
     int i_;
 public:
-    static unsigned dtor_called;
+    static unsigned int dtor_called;

```

Comment:
As [I explained in the issue](https://github.com/microsoft/STL/issues/1389#issuecomment-713860093) (after you submitted this PR) most of this test is mechanically extracted from libc++'s test suite and shouldn't be modified locally because it makes updating from upstream painful. All of these changes are in the extracted part, and should therefore be reverted.